pkg/tuned/tuned.go

```diff
@@ -109,6 +91,35 @@ type wqKey struct {
 	event string // object event type (add/update/delete) or pass the full object on delete
 }
 
+// Constants
+const (
+	// Constants used for instantiating Profile status conditions
```

Comment:
It might be nice to explain the following three lines in this comment, or at least say that they will be set to 1,2,4 respectively

Reply:
A fair point, comments added.  Thank you!